server/graphql/v2/mutation/TierMutations.js

```diff
@@ -0,0 +1,112 @@
+import { GraphQLNonNull } from 'graphql';
+
+import { mustBeLoggedInTo } from '../../../lib/auth';
+import models from '../../../models';
+import { NotFound, Unauthorized } from '../../errors';
+import { getIntervalFromTierFrequency } from '../enum/TierFrequency';
+import { getDecodedId } from '../identifiers';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { getValueInCentsFromAmountInput } from '../input/AmountInput';
+import { TierCreateInput } from '../input/TierCreateInput';
+import { fetchTierWithReference, TierReferenceInput } from '../input/TierReferenceInput';
+import { TierUpdateInput } from '../input/TierUpdateInput';
+import { Tier } from '../object/Tier';
+
+const tierMutations = {
+  editTier: {
+    type: Tier,
+    description: 'Edit a tier.',
+    args: {
+      tier: {
+        type: new GraphQLNonNull(TierUpdateInput),
+      },
+    },
+    async resolve(_, args, req) {
+      mustBeLoggedInTo(req.remoteUser, 'edit a tier');
```

Comment:
This one will also ensure that we verify the scope for people using the API via OAuth

```suggestion
      checkRemoteUserCanUseAccount(req);
```